src/mds/MDSRankQuiesceInterface.cc

```diff
@@ -234,37 +237,58 @@ void MDSRank::quiesce_agent_setup() {
       dout(20) << "parsed root '" << root <<"' as : " << uri->path() << " " << uri->query() << dendl;
     }
 
-    std::lock_guard l(mds_lock);
+    std::chrono::milliseconds quiesce_delay = 0ms;
+    if (auto pit = uri->params().find("delay"); pit != uri->params().end()) {
+      try {
+        quiesce_delay = std::chrono::milliseconds((*pit).has_value ? std::stoul((*pit).value) : 1000);
```

Comment:
i prefer seconds here and below for the `q` as it used to be. These are debug arguments that people are going to type in. It's much more user-friendly, IMO, especially since all other timeouts on the command itself are specified in seconds.

Reply:
additionally, there are already users in the QE team who know about `q=5` which will quiesce in 5 seconds. it'll be awkward to type `?q=5000 --timeout=10`

Reply:
I changed `?q` back to seconds. I would prefer to keep `?delay` in milliseconds to keep the finer resolution and avoid floats.

> These are debug arguments that people are going to type in.

I would think that'd mean the options are for debugging and scripted so "user-friendliness" is not relevant.

Again, my position has been that we should use milliseconds for all kinds of clock durations across the code base because they're relatively future proof (jiffies in linux are 100Hz) and not difficult to use as a human (even for non-developers). Floats are deceptive because they may have fractional parts that are smaller than a jiffie (and therefore not applicable).

Reply:
I accept your argument for the use of ms across the code and ceph settings. Here though, in this CLI, I put coherency of the api above that. You may notice that I truncate all floats to a tenths of a second when rendering.

But I don't mind if you want the delay to be expressed in ms. I'd just like to ask you then put `ms`  somewhere in the name, if you don't mind. This way it won't be ambiguous to users

Reply:
> But I don't mind if you want the delay to be expressed in ms. I'd just like to ask you then put `ms` somewhere in the name, if you don't mind. This way it won't be ambiguous to users

done